Sources/UID2/UID2Manager.swift

```diff
@@ -17,10 +17,9 @@ public final actor UID2Manager {
     /// Enable or Disable Automatic Refresh via RepeatingTimer
     public var automaticRefreshEnabled = true {
         didSet {
-            if automaticRefreshEnabled {
-                timer.resume()
-            } else {
-                timer.suspend()
+            Task {
+                await checkIdentityExpiration()
```

Comment:
Do we need the `checkIdentityExpiration` here? 🤔

Reply:
My thinking is that `automaticRefreshEnabled` could be toggled at any time by the app before or after an identity is stored or reset in the `UID2Manager` so it's a good practice to reset the check jobs.  The existing check jobs will be deleted and the `UID2Manager` will have an opportunity to see if new ones need to be created based on whether or not the `UID2Manager` is currently managing an identity.